docs/tutorials/vehicle_model_tutorial.md

```diff
@@ -0,0 +1,255 @@
+
+You can add more signals to the HVAC system, but ensure they are properties, not commands, which we will discuss in the next section. Signals unrelated to HVAC should not be included in the HVAC interface. As suggested in [1.1 DTDL interfaces](#11-dtdl-interfaces), it is beneficial to segregate interfaces to maintain conciseness and group related components together.
+
+### 1.6 DTDL commands
+
+>[Command:](https://azure.github.io/opendigitaltwins-dtdl/DTDL/v3/DTDL.v3#command) "A Command describes a function or operation that can be performed on any digital twin".
+
+Consider our HVAC digital twin that you created in [1.2 Create HVAC and HMI interfaces](#12-create-hvac-and-hmi-interfaces).
+
+To add properties to the HMI digital twin model, replace the existing content of `hmi.json` with the following:
+
+```json
+[
+  {
+    "@context": ["dtmi:dtdl:context;3"],
+    "@type": "Interface",
+    "@id": "dtmi:sdv:HMI;1",
+    "description": "The Human Machine Interface.",
+    "contents": [
+      {
+        "@type": "Command",
+        "@id": "dtmi:sdv:HMI:ShowNotification;1",
+        "name": "ShowNotification",
+        "description": "Show a notification on the HMI.",
+        "request": {
+          "@id": "dtmi:sdv:HMI:ShowNotification:request;1",
+          "name": "Notification",
+          "displayName": "Notification",
+          "description": "The notification to show on the HMI.",
+          "schema": "string"
+        }
+      }
+    ]
+  }
+]
+```
+
+This introduces a command to our HMI system: Show Notification. Similar to the properties we introduced in the previous section, this command has an `@id` value starting with `dtmi:sdv:HMI`, signifying that this command belongs to the sdv domain and HMI interface.
+
+The `ShowNotification` is not a property. This is because properties reflect the state of a digital twin, while `ShowNotification` is an action that can be performed on this HMI digital twin model which qualifies as a command.
+
+Please see [Command](https://azure.github.io/opendigitaltwins-dtdl/DTDL/v3/DTDL.v3#command) for more information on the command type and the descriptions of each field. Similar to the DTDL interface type, Ibeji mandates the description field. Despite DTDL v3 spec considering the `@id` field for commands as optional, Ibeji requires it. This helps in translating your DTDL files to code.
+
+## 2. DTDL validation
+
+Ensure your digital twin models adhere to the [DTDL v3 spec](https://azure.github.io/opendigitaltwins-dtdl/DTDL/v3/DTDL.v3) before translating your digital twin models to code. Use Ibeji's [DTDL tools](https://github.com/eclipse-ibeji/ibeji/blob/main/dtdl-tools/README.md) to validate your DTDL files.
+
+## 3. Translating DTDL to code
+
+
+You have built a basic reference digital twin model of a vehicle with an HVAC system and HMI using DTDL. In this section, you will use convert this model into Rust code.
+
+1. Create a Rust file called `sdv_v1.rs`

```

Comment:
Create a Rust file that provides meta data from the in-vehicle digital twin model